packages/serverpod/lib/src/database/database_connection.dart

```diff
@@ -634,6 +634,67 @@ Current type was $T''');
       retryIf: retryIf,
     );
   }
+
+  /// For most cases use the corresponding method in [Database] instead.
+  /// If [id] in [TableRow] Already Exist then that row will be replaced by new value
```

Comment:
Added Function to batch insert or update of the list of a Table row.
but if the list of table rows with and without IDs is passed it will not work.